packages/react-table/src/components/TableComposable/Td.tsx

```diff
@@ -63,6 +64,16 @@ export interface TdProps extends BaseCellProps, Omit<React.HTMLProps<HTMLTableDa
     /** Callback on toggling of the expansion */
     onToggle?: OnExpand;
   };
+  favorites?: {
+    /** Whether the corresponding row is favorited */
+    isFavorited: boolean;
+    /** Callback on clicking the favorites button */
+    onFavorite?: OnFavorite;
+    /** The row index */
+    rowIndex?: number;
+    /** Additional props forwarded to the FavoritesCell */
+    props?: any;
```

Comment:
DO you think we should make the name more specific here?

Reply:
To use it you need to wrap it in the `favorites` object which gives it context, e.g.
```
favorites: {
  props: { 'aria-label': 'favs' }
}
```